### ISSUE TITLE
For mobile, the ip information does not come out.

### ISSUE THREAD

**Issue by ilhan7777** [KasRoudra/PyPhisher#57]
When logging in to the url from the PC, Victim Ip information is displayed, but the system does not notice that the login is made when entered from the mobile. (form registration works)

**Comment by KasRoudra** [comment]
@ilhan7777 Does it happen for every template or specific one?

**Comment by KasRoudra** [comment]
@ilhan7777 The funny part is, I coded and tested pyphisher almost entirely on android. In most android it is working perfectly(user feedback). Maybe you should try again with a different browser or a different phone.

**Comment by ilhan7777** [comment]
Hope it works on other phones. I tried another template and it worked fine.
Meanwhile, phone model information can also be added to the captured information. And the ability to send the password to my email address would also be nice.

**Comment by KasRoudra** [comment]
@ilhan7777 Thanks for suggestion. I will consider adding the feature of emailing the id and pass. You can try "66" th option to use a custom site template. But it should have index.php, ip.php, login.php etc

**Comment by ilhan7777** [comment]
I am not competent in these matters. Can you help with adding the login page of a website to the system?

**Comment by ilhan7777** [comment]
https://99designs.com/login

**Comment by ilhan7777** [comment]
Hello, I designed the login page. but I don't know how to adapt it.

**Comment by ilhan7777** [comment]
where are your templates?

**Comment by KasRoudra** [comment]
@ilhan7777 Check `~/.websites` directory

**Comment by ilhan7777** [comment]
thank you so much. I succeeded most of it.

**Comment by ilhan7777** [comment]
I have successfully adapted my own login page, thanks for your help.

My other request is for the ability to send results by mail, I have an idea to help you;

There is a script called AdvPhisher and it had this feature. When I checked a little, I saw that it works integrated with the PhpMailer script:
https://github.com/PHPMailer/PHPMailer

I don't know how to do it, but I hope this is information that will make your job easier. Thanks.
